test/integration/scenarios/chart-full-setup/Taskfile.yaml

```diff
@@ -125,4 +127,4 @@ tasks:
     - task: setup.post
     - task: test.preflight
     - task: test.core
-    - task: test.playwright.core
\ No newline at end of file
+    - task: test.playwright.core
```

Comment:
Not related to this PR, really, but why do we frequently get  git trying to stage changes with the final character in the file?  is this some linter nonsense?